media/css/mozorg/about-base.less

```diff
@@ -131,6 +131,10 @@ html[dir="rtl"] {
         margin-right: 0;
         margin-left: 490px;
     }
+    #main-content .intro {
+        float: right;
+
```

Comment:
Style consistency: no empty line in the middle of the rule, empty line before the rule.